homeassistant/components/transmission/services.yaml

```diff
@@ -9,6 +9,12 @@ add_torrent:
       example: http://releases.ubuntu.com/19.04/ubuntu-19.04-desktop-amd64.iso.torrent
       selector:
         text:
+    download_path:
+      required: false
+      default: None
```

Comment:
```suggestion
```

Reply:
Do I need to make this commit myself or is this automatically applied?